src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
Examples can prove nothing. I just illustrated a good technique, which can save some time if, as any other good habit, practised regularly. The web contains many discussions on this matter, which you can easily read.

> ...so give it back as it was.

Why?

> If you haven't noticed, inkludes in new code goes in a certain order and visually divided into groups.

The thing is this order is the most useless (except of a purely random) order.